index.bs

```diff
@@ -359,8 +384,12 @@ refuse incoming connections that lack these parameters.
 
 * The [[!RFC7301|ALPN]] used must be "osp".
 * The [[!RFC6066|server_name extension]] must be set to the following `host_name`:
```

Comment:
Let's leave updates to the server name extension out of this PR as here is an active discussion in #275 on this.

Reply:
I'm trying to understand if it's feasible to consider these independently. As far as I can tell: if SNI is used to determine the server's identity, then this value should also be used to validate the server identity of the TLS certificate (`dNSName` or `CN`).

Next to that, #275 seems to have an issue with the `_openscreen._udp.local` post-fix rather than the `<fp>` or `<sn>` prefix. As these names are used for connection and no longer for mDNS discovery it also seems feasible to just use `<sn>.local` as server identity.

References: [RFC2818](https://www.rfc-editor.org/rfc/rfc2818.html#section-3.1) & [RFC6066](https://datatracker.ietf.org/doc/html/rfc6066#section-3)